plugins/modules/cloud/opennebula/one_vm.py

```diff
@@ -1282,6 +1283,29 @@ def resume_vms(module, client, vms):
     return changed
 
 
+def release_vm(module, client, vm):
+    vm = client.vm.info(vm.ID)
+    changed = False
+
+    state = vm.STATE
+    if state != VM_STATES.index('HOLD'):
+        module.fail_json(msg="Cannot perform action 'release' because VM is not in state 'HOLD'.")
+    else:
+        changed = True
+
+    if changed and not module.check_mode:
+        client.vm.action('release', vm.ID)
```

Comment:
I think you want to have:
```suggestion
        client.vm.action('release', vm.ID)

    return changed
```